homeassistant/core.py

```diff
@@ -293,8 +293,9 @@ async def async_start(self) -> None:
             return
 
         self.state = CoreState.running
-        _async_create_timer(self)
+        self.bus.async_fire(EVENT_CORE_CONFIG_UPDATE)
```

Comment:
Should we also fire a config update event for the three changes in `async_stop`?

Reply:
We're stopping and so don't handle any event listeners anymore.

Reply:
Right.